module/Application/src/Module.php

```diff
@@ -50,7 +52,8 @@ public function onBootstrap(MvcEvent $e): void
 
         $locale = $this->determineLocale($e);
 
-        $mvcTranslator = $e->getApplication()->getServiceManager()->get(MvcTranslator::class);
+        $serviceManager = $e->getApplication()->getServiceManager();
+        $mvcTranslator = $serviceManager->get(MvcTranslator::class);
```

Comment:
What is the reason for splitting this?

Reply:
These were remains from an earlier version of the code which reused `$serviceManager`